ops/daily-dev-snapshot.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash

```

Comment:
Let's rename it to `snapshot-disk-and-cleanup.sh`. There is nothing inside the script that is doing anything "daily" ;) The "daily" aspect comes from the outer context, our CI. Otherwise - looks great! 👍  But I need to test it first before we merge.